OSDOCS-5637-installer: Documented the installer bug fix release notes

dfitzmau opened the pull request:
[OSDOCS-5637](https://issues.redhat.com/browse/OSDOCS-5637)

Version(s):
4.13

Link to docs preview:
[Preview](https://58496--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-bug-fixes)

QE review:
- [ ] QE has approved this change.

dfitzmau commented:
/label peer-review-needed

jeana-redhat commented on release_notes/ocp-4-13-release-notes.adoc:
```diff
@@ -967,6 +967,16 @@ With this release, replacement control plane nodes are assigned to the correct i
+* Previously, SSH access to bootstrap and cluster nodes failed when the bastion host ran in the same VPC network as the cluster nodes. Additionally, this configuration caused SSH access from the temporary bootstrap node to the cluster nodes to fail. These issues are now fixed by updating the the IBM Cloud `SecurityGroupRules` to support SSH traffic between the temporary bootstrap node and cluster ...
+
+* Previously, if you configured the rendezvous IP to the IP address of the host that has a `role` parameter set to `worker` and you generated an ISO image, the Agent-based installer would fail to install the cluster. Now, when you attempt to generate an ISO image based on this configuration, you will receive a validation failure message. On receiving this message, you must update the `rendezvousI ...
+
+OCPBUGS-2088
```
```suggestion
```

dfitzmau replied:
Woops! Removed.

jeana-redhat commented on release_notes/ocp-4-13-release-notes.adoc:
```diff
@@ -967,6 +967,16 @@ With this release, replacement control plane nodes are assigned to the correct i
 [id="ocp-4-13-installer-bug-fixes"]
 ==== Installer
 
+* Previously, DNS records that the installation program created were not removed when uninstalling a private cluster. With this update, the installation program now correctly removes these DNS records. (link:https://issues.redhat.com/browse/OCPBUGS-7973[OCPBUGS-7973]
```
```suggestion
* Previously, DNS records that the installation program created were not removed when uninstalling a private cluster. With this update, the installation program now correctly removes these DNS records. (link:https://issues.redhat.com/browse/OCPBUGS-7973[*OCPBUGS-7973*])
```

jeana-redhat commented on release_notes/ocp-4-13-release-notes.adoc:
```diff
@@ -967,6 +967,16 @@ With this release, replacement control plane nodes are assigned to the correct i
 
+* Previously, DNS records that the installation program created were not removed when uninstalling a private cluster. With this update, the installation program now correctly removes these DNS records. (link:https://issues.redhat.com/browse/OCPBUGS-7973[OCPBUGS-7973]
+
+* Previously, the bare metal installer-provisioned infrastrature used port 80 for providing images to the Baseboard Management Controller (BMC) and the deployment agent. Security risks could exist with port 80, because this port is commonly chosen for internet communcations. The bare metal installer-provisioned infrastrature now used port 6180 for serving images that are used by the `metal3` pod  ...
```
```suggestion
* Previously, the bare metal installer-provisioned infrastructure used port 80 for providing images to the Baseboard Management Controller (BMC) and the deployment agent. Security risks could exist with port 80, because this port is commonly chosen for internet communications. The bare metal installer-provisioned infrastructure now uses port 6180 for serving images that are used by the `metal3` pod on deployed clusters. (link:https://issues.redhat.com/browse/OCPBUGS-8511[*OCPBUGS-8511*])
```

dfitzmau replied:
I took the note verbatim. Well spotted. I've learned to read over them in detail before creating a PR. Thanks!

jeana-redhat commented on release_notes/ocp-4-13-release-notes.adoc:
```diff
@@ -967,6 +967,16 @@ With this release, replacement control plane nodes are assigned to the correct i
+
+* Previously, the bare metal installer-provisioned infrastrature used port 80 for providing images to the Baseboard Management Controller (BMC) and the deployment agent. Security risks could exist with port 80, because this port is commonly chosen for internet communcations. The bare metal installer-provisioned infrastrature now used port 6180 for serving images that are used by the `metal3` pod  ...
+
+* Previously, SSH access to bootstrap and cluster nodes failed when the bastion host ran in the same VPC network as the cluster nodes. Additionally, this configuration caused SSH access from the temporary bootstrap node to the cluster nodes to fail. These issues are now fixed by updating the the IBM Cloud `SecurityGroupRules` to support SSH traffic between the temporary bootstrap node and cluster ...
```
Is `SecurityGroupRules` an "object"? Or some other type of noun that I can't think of? :wink: Could also maybe say "the IBM Cloud security group rules"

```suggestion
* Previously, SSH access to bootstrap and cluster nodes failed when the bastion host ran in the same VPC network as the cluster nodes. Additionally, this configuration caused SSH access from the temporary bootstrap node to the cluster nodes to fail. These issues are now fixed by updating the IBM Cloud `SecurityGroupRules` to support SSH traffic between the temporary bootstrap node and cluster nodes, and to support SSH traffic from a bastion host to cluster nodes on the same VPC network. Log and debug information can be accurately collected for analysis during installer-provisioned infrastructure failure.(link:https://issues.redhat.com/browse/OCPBUGS-8035[*OCPBUGS-8035*])
```

dfitzmau replied:
Updated

jeana-redhat commented on release_notes/ocp-4-13-release-notes.adoc:
```diff
@@ -967,6 +967,16 @@ With this release, replacement control plane nodes are assigned to the correct i
+
+* Previously, SSH access to bootstrap and cluster nodes failed when the bastion host ran in the same VPC network as the cluster nodes. Additionally, this configuration caused SSH access from the temporary bootstrap node to the cluster nodes to fail. These issues are now fixed by updating the the IBM Cloud `SecurityGroupRules` to support SSH traffic between the temporary bootstrap node and cluster ...
+
+* Previously, if you configured the rendezvous IP to the IP address of the host that has a `role` parameter set to `worker` and you generated an ISO image, the Agent-based installer would fail to install the cluster. Now, when you attempt to generate an ISO image based on this configuration, you will receive a validation failure message. On receiving this message, you must update the `rendezvousI ...
```
```suggestion
* Previously, if you configured the rendezvous IP to the IP address of the host that has a `role` parameter set to `worker` and you generated an ISO image, the Agent-based installer would fail to install the cluster. Now, when you attempt to generate an ISO image based on this configuration, you receive a validation failure message. On receiving this message, you must update the `rendezvousIP` field in the `agent-config.yaml` file to use the IP of a host with the `master` role. (link:https://issues.redhat.com/browse/OCPBUGS-2088[*OCPBUGS-2088*]) 
```

dfitzmau replied:
Updated.

jeana-redhat reviewed (commented):
Little bits here and there

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

dfitzmau commented:
Hi @jeana-redhat . Thanks for the review. I updated the content based on your feedback. 

Re: `SecurityGroupRules`, you are correct. I updated the term to `security group rules` based on the term use in the following article:

[About IBM security groups](https://cloud.ibm.com/docs/security-groups?topic=security-groups-about-ibm-security-groups#:~:text=An%20IBM%20Cloud%C2%AE%20security,known%20as%20security%20group%20rules.)

dfitzmau commented:
/label merge-review-needed

opayne1 reviewed (commented):
/lgtm
